src/views/v2/tickets/item/quantity/add.php

```diff
@@ -14,8 +14,8 @@
  *
  * @version TBD
  *
- * @var $this Tribe__Tickets__Editor__Template
- * @var Tribe__Tickets__Ticket_Object $ticket The ticket object.
+ * @var	Tribe__Tickets__Editor__Template $this   Template object.
```

Comment:
:no_entry_sign: Spaces must be used for mid-line alignment; tabs are not allowed

<sub>hash: 897ca682298ced726f40e92980cc6133</sub>